Ghost_in_the_Shell/client_folder_react_app/ethisim/src/Constants/Config.js

```diff
@@ -1,3 +1,3 @@
-const baseURL = 'http://localhost:8000';
-
+//const baseURL = 'http://localhost:8000';
+const baseURL = 'http://127.0.0.1:8000/api/';
 export { baseURL };
```

Comment:
You can't change the baseURL, that will break every other HTTP request in the repository